src-tauri/src/remote/auth.rs

```diff
@@ -222,38 +194,12 @@ pub fn setup() -> Result<(AppStatus, Option<User>), ()> {
             let error = user_result.err().unwrap();
             warn!("auth setup failed with: {}", error);
             match error {
-                RemoteAccessError::FetchError(_) => {
-                    return Ok((AppStatus::ServerUnavailable, None))
-                }
-                _ => return Ok((AppStatus::SignedInNeedsReauth, None)),
+                RemoteAccessError::FetchError(_) => return (AppStatus::ServerUnavailable, None),
```

Comment:
It's unacceptable to eat errors like this, as in the fetch error should be a part of the error. 

I do realise this is what we were doing before

Reply:
That's what I mean, we should be returning actual errors